<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>purchase</class>
 <widget class="QDialog" name="purchase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
	background-color: rgb(34, 40, 49);
	
}



QDialog
{
	background-color: rgb(238, 238, 238);
}



QLabel
{
	background-color:rgb(238, 238, 238);
	color: rgb(34, 40, 49);
	font: 11pt &quot;DejaVu Sans Mono&quot;;
}

QLineEdit
{

	color:rgb(238, 238, 238);
	background-color: rgb(34, 40, 49);
	border-radius: 9px;
	font: 9pt &quot;DejaVu Sans Mono&quot;;

}


QPushButton
{
	border-radius: 12px;
	background-color: rgb(255, 211, 105);
	color: rgb(34, 40, 49);
	font: 700 11pt &quot;DejaVu Sans Mono&quot;;
	
	
}




</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>0</y>
     <width>841</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 18pt &quot;Ubuntu&quot;;
background-color: rgb(34, 40, 49);
color:rgb(238, 238, 238);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Purchase</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line2_2">
   <property name="geometry">
    <rect>
     <x>-250</x>
     <y>170</y>
     <width>1300</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(57, 62, 70);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="errorNumber">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);
font: 9pt &quot;DejaVu Sans Mono&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberIn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantityIn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="codeIn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="errorCode">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);
font: 9pt &quot;DejaVu Sans Mono&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Code</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushproduct">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="Line" name="line2_3">
   <property name="geometry">
    <rect>
     <x>-310</x>
     <y>320</y>
     <width>1300</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(57, 62, 70);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="errorQuantity">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);
font: 9pt &quot;DejaVu Sans Mono&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="purchaseInfo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>551</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 500 11pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);
text-align: left;</string>
   </property>
   <column>
    <property name="text">
     <string>Product Code</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Product Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Unit Price (Rs.)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Grand Total </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>710</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Tax (13%) </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>750</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Net-Total (Rs.)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="grandtotalOut">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>670</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="taxOut">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>710</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="nettotalOut">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>750</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="getBill">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>590</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Bill</string>
   </property>
  </widget>
  <widget class="Line" name="line2_4">
   <property name="geometry">
    <rect>
     <x>-170</x>
     <y>640</y>
     <width>1300</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(57, 62, 70);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="purchaseSucessful">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>790</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);
font: 9pt &quot;DejaVu Sans Mono&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>numberIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>errorNumber</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>codeIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>errorCode</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quantityIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>errorQuantity</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>purchaseSucessful</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>798</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
